backend/api/views/main.py

```diff
@@ -55,3 +56,71 @@ def create_person():
     return create_response(
         message=f"Successfully created person {new_person.name} with id: {new_person.id}"
     )
+
+
+# POST request for a new mentor profile
+@main.route("/mentor", methods=["POST"])
+def create_mentor_profile():
+    data = request.json
+    validate_data = MentorForm.from_json(data)
+
+    # Used WTForms and WTForms-JSON to validate that there exists values
+    if not validate_data.validate():
+        msg = ", ".join(validate_data.errors.keys())
+        return create_response(status=422, message="Missing fields " + msg)
+
+    new_mentor = MentorProfile(
+        name=data["name"],
+        professional_title=data["professional_title"],
+        linkedin=data["linkedin"],
+        website=data["website"],
+        picture=data["picture"],
+        languages=data["languages"],
+        specializations=data["specializations"],
+        offers_in_person=data["offers_in_person"],
+        offers_group_appointments=data["offers_group_appointments"],
+    )
+
+    # If option field Biography is passed
+    if "biography" in data:
+        new_mentor.biography = data["biography"]
```

Comment:
instead of doing checks like this you can do something similar to @AzureQDragon where you set it to None if it doesn't exist- OR even better, you can pass data.get(biography) into the constructor to extra clean your code
```suggestion
new_mentor.biography = data.get("biography")
```
since dict.get defaults to None if nothing's there